packages/buefy-next/package.json

```diff
@@ -59,6 +59,7 @@
     "@rollup/plugin-node-resolve": "^15.2.3",
     "@rollup/plugin-terser": "^0.4.4",
     "@testing-library/jest-dom": "^6.1.4",
+    "@types/jest": "^29.5.12",
```

Comment:
I do not think we need to install type definitions for jest because we won't use it with TypeScript.

Reply:
If your concern is about the type of `jest.fn`, please see [this comment](https://github.com/ntohq/buefy-next/pull/277/files#r1759723222).